data/party/noelle.lua

```diff
@@ -103,18 +98,27 @@ function character:init()
     self.flags = {
         ["iceshocks_used"] = 0,
         ["boldness"] = -12,
+        ["auto_title"] = true,
         ["weird"] = false
     }
-end
-
-function character:getTitle()
-    if self:checkWeapon("thornring") then
-        return "LV"..self.level.." Ice Trancer\nReceives pain to\nbecome stronger."
-    elseif self:getFlag("iceshocks_used", 0) > 0 then
-        return "LV"..self.level.." Frostmancer\nFreezes the enemy."
-    else
-        return "LV1 "..self.title
-    end
+    
+    -- Display level (saved to the save file)
+    -- Default title / class (saved to the save file)
+    Game.stage.timer:everyInstant(1/30, function()
```

Comment:
I kinda wonder if `0` works too. Still reads stupid though but IDK how to fix it